pipelines/batch/run-batch-local.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
@bashir2 this is a new script I added for someone to run the batch pipeline locally

Reply:
Thanks for adding this; can you please remind me how we intend to use this? I think the main way to run e2e test locally was through running `cloud-build-tool`; so is the idea here that people use that to bring up required docker images but during the development of the batch pipeline, use this script to run the batch pipeline faster?

nit: For consistency let's use `_` instead of '-' in file names, as much as possible. I think in scripts the only other exception is `batch-mode-validation.sh` which is also renamed in this PR. So we can fix both names right here in this PR.

Reply:
> I think the main way to run e2e test locally was through running cloud-build-tool; so is the idea here that people use that to bring up required docker images but during the development of the batch pipeline, use this script to run the batch pipeline faster?

Yes, the main way to run e2e tests locally is through `cloud-build-tool`. The use case for this script is during development. If a developer has the server up, and the code compiled, there's no need for them to run the e2e-test via the `cloud-build-tool`. Instead, they can run `run`_batch_local.sh` to build and run the batch pipeline